src/main/java/edu/greenblitz/robotName/Robot.java

```diff
@@ -50,6 +60,17 @@ public void robotPeriodic() {
         RoborioUtils.updateCurrentCycleTime();
     }
 
+    private void initializeAutonomousBuilder(){
+        AutoBuilder.configureHolonomic(
+                SwerveChassis.getInstance()::getRobotPose,
+                SwerveChassis.getInstance()::resetChassisPose,
+                SwerveChassis.getInstance()::getRobotRelativeChassisSpeeds,
+                SwerveChassis.getInstance()::moveByRobotRelativeSpeeds,
+                ChassisConstants.PATH_FOLLOWER_CONFIG,
+                () -> FMSUtils.getAlliance() == DriverStation.Alliance.Red,
```

Comment:
Why is this specific to the red alliance?

@NoamRosenberg08 says blue should the default

Reply:
the boolean is "is flipped" it will be true if our alliance is red